core/common/src/ipc/IpcSession.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module IpcSocket
+ */
+
+/** @internal */
+export interface RPC {
```

Comment:
This name seems a bit confusing, _especially_ considering it's only used for RPC over IPC... 

That being said, I think I understand why this is hard to name - everything applicable seems a bit overloaded by actual RPC types...